Common/Securities/Forex/Forex.cs

```diff
@@ -47,7 +47,8 @@ public Forex(SecurityExchangeHours exchangeHours, Cash quoteCurrency, Subscripti
                 new ImmediateSettlementModel(),
                 Securities.VolatilityModel.Null,
                 new SecurityMarginModel(50m),
-                new ForexDataFilter()
+                new ForexDataFilter(),
+                new AdjustedPriceVariationModel()
```

Comment:
I think this should be done because backtests will now be more accurate and behavior same as in live trading, although there will be some small difference in overall results (may have to check and update some regression stats as well)